### PR TITLE
add support for django admin filters

### PR DESCRIPTION
**Pull request by timur-orudzhov**

### CODE REVIEW

File: adminsortable/templates/adminsortable/change_list.html [hunk 1]
```diff
@@ -29,6 +29,23 @@
 <script src="{% static 'adminsortable/js/jquery.ui.touch-punch.min.js' %}"></script>
 {% include 'adminsortable/csrf/jquery.django-csrf.html' with csrf_cookie_name=csrf_cookie_name %}
 <script src="{% static 'adminsortable/js/admin.sortable.js' %}"></script>
+
+<script type="text/javascript">
+(function($) {
+    $(document).ready(function($) {
+        var url = window.location.href;
+        var urlParts = url.split('?');
+        var cnangeListUrl = $('a#return-to-changelist').attr('href')
```
Comment (alsoicode) [inline adminsortable/templates/adminsortable/change_list.html]: You have a small typo here. I think you meant `changeListUrl` instead of `cnangeListUrl`

Comment (timur-orudzhov) [reply adminsortable/templates/adminsortable/change_list.html]: yep. thank you

Comment (timur-orudzhov) [reply adminsortable/templates/adminsortable/change_list.html]: @alsoicode what do you think about this pr ? is it useful ?

**Comment by alsoicode** [comment]
Version 2.1.5 has been released to PyPi. Thanks!

**Pull request merged**